README.md

```diff
@@ -144,26 +113,31 @@ You can deploy this project using:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/collins-kiprotich/math-magician/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
 
-If you like this project...
+If you like this project please give it a star
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank code reviewers at microverse for their incredible help
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 📝 License <a name="license"></a>
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 fix the style for professional documentation.

![image](https://user-images.githubusercontent.com/58642949/230369859-24b08269-a9cd-4b78-8881-2905c71568fe.png)